src/components/ContactCard/ContactCard.jsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { translate } from "cozy-ui/react/I18n";
+import Modal, { ModalContent } from "cozy-ui/react/Modal";
+import ContactIdentity from "./ContactIdentity";
+import ContactFields from "./ContactFields";
+import contactPropTypes from "../ContactPropTypes";
+
+const HeaderActions = () => null;
+
+export const supportedFieldsInOrder = [
+  "phone",
+  "email",
+  "address",
+  "cozy",
+  "company",
+  "birthday",
+  "note"
+].reverse();
+
+export const contactToFieldList = contact =>
```

Comment:
As you get contact as a parameter, we must clearify the function name: `getTypeValuesFrom(contact)` for instance. Or something like `getFieldList(contact)`.